common/workunit/workunit.cpp

```diff
@@ -4424,7 +4425,8 @@ class CEnvironmentClusterInfo: implements IConstWUClusterInfo, public CInterface
                     throw MakeStringException(WUERR_MismatchClusterSize,"CEnvironmentClusterInfo: Thor cluster can not have 0 slave processes");
                 unsigned slavesPerNode = thor.getPropInt("@slavesPerNode", 1);
                 unsigned channelsPerSlave = thor.getPropInt("@channelsPerSlave", 1);
-                unsigned ts = nodes * slavesPerNode * channelsPerSlave;
+                numberOfSlaveLogs = nodes * slavesPerNode;
+                unsigned ts = numberOfSlaveLogs * channelsPerSlave;
```

Comment:
minor - I probably would have left the original line:
unsigned ts = nodes * slavesPerNode * channelsPerSlave;